<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_rename_files</class>
 <widget class="QDialog" name="Dialog_rename_files">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rename photos</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>590</y>
     <width>331</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>721</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>If alphabetical or date order matters in case you want to autonumber your images, then make sure that you load your images in the right order. This is not a problem for the Source folder here but can be a problem if you use a set of images from the Main screen.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>580</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Name:  Prefix_Suffix.extension</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>761</width>
     <height>451</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="simple">
    <attribute name="title">
     <string>Simple</string>
    </attribute>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>270</y>
       <width>221</width>
       <height>141</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>File extension:</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>191</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QRadioButton" name="radioButton_fileext_asis">
         <property name="text">
          <string>Leave as is</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_fileext_tolower">
         <property name="text">
          <string>Make lowercase</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_fileext_toupper">
         <property name="text">
          <string>Make uppercase</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Select rename options:</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>421</width>
       <height>121</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>381</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>In case your chosen options lead to duplicate names, the tool will automatically autonumber the images.
Please specify how the tool should do this.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>391</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>No. of digits</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_digits">
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_startcount">
         <property name="toolTip">
          <string>When counting from the 1st image, the counting runs from 1 to n, like 01, 02, 03, etc.  When starting on the second image the counting runs from 00, 01, 02, etc.</string>
         </property>
         <item>
          <property name="text">
           <string>Start counting on 1st image</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Start counting on 2nd image</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>371</width>
       <height>231</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>342</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>10</number>
       </property>
       <item row="2" column="2">
        <widget class="QLineEdit" name="suffix_string"/>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="radioButton_suffix_datetime">
         <property name="text">
          <string>Date_time</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Suffix:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="radioButton_suffix_string">
         <property name="text">
          <string>String</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QComboBox" name="comboBox_suffix_datetime">
         <item>
          <property name="text">
           <string>YYYYMMDDHHMMSS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>YYYYMMDD_HHMMSS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>YYYMMDD-HHMMSS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>YYYY_MM_DD_HH_MM_SS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>YYYY-MM-DD-HH-MM-SS</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QComboBox" name="comboBox_suffix_date">
         <item>
          <property name="text">
           <string>YYYYMMDD</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>YYYY_MM_DD</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>YYYY-MM-DD</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QRadioButton" name="radioButton_suffix_date">
         <property name="text">
          <string>Date</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="radioButton_suffix_donotuse">
         <property name="text">
          <string>Do not use</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QRadioButton" name="radioButton_model">
         <property name="text">
          <string>Camera model</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>${Exif:Model}</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QRadioButton" name="radioButton_orgfilename">
         <property name="text">
          <string>Original filename</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>${filename}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>321</width>
       <height>141</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>303</width>
        <height>130</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>10</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Prefix:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBox_prefix_date">
         <item>
          <property name="text">
           <string>YYYYMMDD</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>YYYY_MM_DD</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>YYYY-MM-DD</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="radioButton_prefix_datetime">
         <property name="text">
          <string>Date_time</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="radioButton_prefix_date">
         <property name="text">
          <string>Date</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBox_prefix_datetime">
         <item>
          <property name="text">
           <string>YYYYMMDDHHMMSS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>YYYYMMDD_HHMMSS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>YYYMMDD-HHMMSS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>YYYY_MM_DD_HH_MM_SS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>YYYY-MM-DD-HH-MM-SS</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="radioButton_prefix_string">
         <property name="text">
          <string>String</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="prefix_string"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>321</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Note: Some combinations of prefix and suffix are not allowed. Upon selection of the prefix, only the allowed suffixes are enabled.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="doityourself">
    <attribute name="title">
     <string>Do it yourself</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>721</width>
       <height>311</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="horizontalSpacing">
       <number>50</number>
      </property>
      <item row="3" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>d04 : date_time (YYYY_MM_DD_HH_MM_SS)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>d01 : date_time (YYYYMMDDHHMMSS)</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>d07 : date (YYYY_MM_DD)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>d05 : date_time (YYYY-MM-DD-HH-MM-SS)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>d06 : date (YYYYMMDD)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>d03 : date_time (YYYYMMDD-HHMMS)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>do2 : date_time (YYYYMMDD_HHMMSS)</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>d08 : date (YYYY-MM-DD)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="s01">
        <property name="toolTip">
         <string>You can use this string only in combination with one of the other options</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>s02 : Camera model (${Exif:Model})</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>s03 : Original filename (${filename})</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>s05 : Iso value (${iso})</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>s04 : Author (${author})</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>701</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="button_rename_info">
      <property name="text">
       <string>Info</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="LineEdit_rename_source_folder">
      <property name="toolTip">
       <string>If you select a source folder all images in that folder will be renamed. Note that also images in subfolders will be renamed.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_rename_browse">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Source folder:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_rename_files</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_rename_files</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
